torch/csrc/autograd/engine.h

```diff
@@ -94,13 +94,22 @@ struct GraphTask {
     return exec_info_.empty();
   }
 
-  GraphTask(bool keep_graph, bool grad_mode, int reentrant_depth)
+  // Whether or not to stop execution for this GraphTask when an error is
+  // encountered.
```

Comment:
What does it mean to "stop" execution for the GraphTask? You can't immediately shut down all the threads; do you also clear out all the ready queues in this case? (What if someone reinitiates a backwards?)